app/views/file_attachments/confirm_delete.html.erb

```diff
@@ -14,7 +14,7 @@
       method: :delete,
       multipart: true,  
       data: {
-        "modal-action": "confirm-delete",
+        "modal-action": "confirmDelete",
```

Comment:
Again I would split this out into a separate, tidy-up commit, rather than bundle it in with the "Add delete confirmation functionality". Not essential but it is a good habit to get into.

Reply:
Thanks Chris, having discussed this with Ben I think I will leave it this time but will definitely start doing it in my future PRs 👍